Example/Tests/CISAdAnalyticsTestDouble.swift

```diff
@@ -9,48 +9,88 @@
 import ConvivaSDK
 import Foundation
 
-class CISAdAnalyticsTestDouble: NSObject, CISAdAnalyticsProtocol {
+class CISAdAnalyticsTestDouble: NSObject, CISAdAnalyticsProtocol, TestDoubleDataSource {
     func isAirPlaying() -> Bool {
         false
     }
 
     func setAdInfo(_ adInfo: [AnyHashable: Any]) {
+        var args = [String: String]()
+        adInfo.forEach { key, value in args[String(describing: key)] = String(describing: value) }
```

Comment:
Following the previous comment. If you want to keep this I would suggest to extract this same line into an extension:

```swift
private extension Dictionary where Key == AnyHashable, Value == Any {
    func fooBar() -> [String: String] {
        var args = [String: String]()
        forEach { key, value in
            args[String(describing: key)] = String(describing: value)
        }
        return args
    }
}
```

Reply:
https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/pull/69/commits/dfe865e8ed81d8fb85e8fb25b6293b661c18ea60

Reply:
https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/pull/69/commits/c62416281bf57b807e54259b865f78982a1c1ab6